overlord/ifacestate/ifacestate.go

```diff
@@ -44,6 +44,11 @@ type ErrAlreadyConnected struct {
 	Connection interfaces.ConnRef
 }
 
+const (
+	ConnectTask       = state.TaskSetEdge("connect-task")
```

Comment:
`download-and-checks-done` is named `DownloadAndChecksDoneEdge`. Maybe we should use `ConnectPrepareDoneEdge` and `ConnectHooksDoneEdge`.

Reply:
Done.